### ISSUE TITLE
When remote actions from NVDA menu cause errors, a message should be issued to the user

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#17947]
### Steps to reproduce:
* Enable Remote Access
* Set up a remote connection but do not connect remote machine
* Press NVDA menu -> Tools -> Remote -> Push clipboard
* Press NVDA menu -> Tools -> Remote -> Send Ctrl+Alt+Del

### Actual behavior:
* For Send Ctrl+Alt+Del: an error is logged and the user gets no error message.
* For Push clipboard: The message "No one else is connected" is almost not hearable; it is interrupted by the new focus after the menu is closed.
### Expected behavior:
When the menu closes, it is expected that the action has succeeded. For example "Revert to saved configuration", or "Mute remote" have also their messages interrupted, but that's not considered a problem when the action succeeds.

But when the action fails, a message should be issued to the user through a message box. `ui.message` is not acceptable as a feedback for menu actions since it can be overridden by new focus event. And also, because the menu is a visual GUI; a sighted person would expect a standard feedback to menu actions, i.e. no feedback when success (e.g. File -> Save), but dialog box on failure (e.g. Error message when trying File -> Save after having unplugged an USB stick from where the file was opened).

### NVDA logs, crash dumps and other attachments:
When sending Ctrl+Alt+Del, the following error is logged:
```
ERROR - _remoteClient.client.RemoteClient.sendSAS (10:35:47.144) - MainThread (14028):
No leader transport to send SAS
```
#### NVDA installed/portable/running from source:
From source
#### NVDA version:
Last beta 9833d781273eeb6f4891e2cba8e110cbf4e725af
#### Windows version:
Windows 10 22H2 (AMD64) build 19045.5608
#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Not tested
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
No
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested
